clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/description/UpsertAmazonLoadBalancerClassicDescription.java

```diff
@@ -176,5 +178,26 @@ public String getSslCertificateId() {
     public void setSslCertificateId(String sslCertificateId) {
       this.sslCertificateId = sslCertificateId;
     }
+
+    public List<CookiePolicy> getCookiePolicies() {
+      return cookiePolicies;
+    }
+
+    public void setCookiePolicies(List<CookiePolicy> cookiePolicies) {
+      this.cookiePolicies = cookiePolicies;
+    }
+
+    @Data
```

Comment:
`@Data` annotation, but all of the properties are defined as `public`? What's the point?

Reply:
re: no test - Oh yeah there will be tests, wanted to see if this was good direction from folks.